packages/workflow-core/src/lib/index.ts

```diff
@@ -1,14 +1,46 @@
-export { createWorkflow } from './create-workflow';
-export { HttpError } from './errors';
-export {
-  Error,
-  Errors,
+export { Error, Errors } from './types';
+export type {
   WorkflowEvent,
   WorkflowEventWithoutState,
   WorkflowOptions,
   WorkflowRunnerArgs,
   WorkflowContext,
+  TCreateWorkflow,
+  ChildWorkflowCallback,
+  ChildPluginCallbackOutput,
+  ChildToParentCallback,
+  SerializableTransformer,
+  WorkflowExtensions,
+  Workflow,
+  ObjectValues,
 } from './types';
-export { StatePlugin } from './plugins/types';
-export { PluginAction } from './plugins/types';
-export { ExtensionRunOrder } from './plugins/types';
+export type {
```

Comment:
Ensure all exported items are really types. If they are, they are probably supposed to be in a types file.